content/ja/website-design.md

```diff
@@ -78,30 +78,30 @@ eSoliaのWebデザインチームは、ゼロから日本のサイトを作成
 
 #### 静的・動的CMS
 
-eSoliaは今貴方が見ている静的なサイトのみならず、Wordpress・TYPO3のようなCMSベースのサイトの展開も可能です 動的CMSでホストされているものと静的Webサイトに関する考慮事項を次の表に示します。
+今あなたが見ている静的なサイトのみならず、Wordpress・TYPO3のような動的CMSベースのサイトの展開も可能です 動的CMSでホストされているものと静的Webサイトに関して考慮されるべき事項を次の表に示します。
 
 _項目_  |静的    | 動的CMS
 ----------|----------|------
 _パフォーマンス_    |最高       |低
-_セキュリティ_       |最高    |低
-_インフラストラクチャ_       |シンプル    |複雑
-_バックアップ_     |容易     |困難
+_セキュリティ_       |高    |低
+_インフラストラクチャ_       |単純    |複雑
+_バックアップ_     |容易     |難しい
 _データ駆動化_  |いいえ     |はい
 _コンテンツ更新_  |ファイル単位    |ダッシュボード
-_視覚スタイル_   |汎用    |テーマ
-_アップグレード_    |任意   |継続的・脆弱
+_視覚スタイル_   | どの様なものも   |テーマ別
+_アップグレード_    |任意   |継続的かつ脆弱
 
```

Comment:
The table here morphed away from the meaning of what I meant when I wrote the English, so I corrected it in a commit after this one. @hmaezono